styles/fonts/icomoon.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
SOMETHING ABOUT ICOMOON ISNT WORKING FOR ME... this weekend i might switch to font awesome instead...